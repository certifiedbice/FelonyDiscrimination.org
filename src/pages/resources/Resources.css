main div h3 {
    margin: 20px 0 10px 0;
    text-align: left;
}