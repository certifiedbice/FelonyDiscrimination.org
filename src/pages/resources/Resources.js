import React from 'react';
import {Link} from 'react-router-dom';
import './Resources.css';

export default function Resources(){
	return(
		<div>
			<h2>Resources Page</h2>
            <Link to='/needtoknow'>Need to Know</Link>
		</div>
	);
}