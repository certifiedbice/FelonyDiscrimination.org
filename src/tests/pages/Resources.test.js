import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import Resources from '../../pages/resources/Resources';

it('renders without crashing',()=>{
    const div=document.createElement('div');
    ReactDOM.render(<BrowserRouter><Resources/></BrowserRouter>,div);
});

it('renders correctly',()=>{
    const tree=renderer
        .create(<BrowserRouter><Resources/></BrowserRouter>)
        .toJSON();
        expect(tree).toMatchSnapshot();
});