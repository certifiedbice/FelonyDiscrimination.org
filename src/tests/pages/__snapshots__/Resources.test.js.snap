// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    Resources Page
  </h2>
  <a
    href="/needtoknow"
    onClick={[Function]}
  >
    Need to Know
  </a>
</div>
`;
